.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+repos:
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: v0.0.254
+    hooks:
+      - id: ruff
+        args:
+          - --fix
```

Comment:
I believe it's also currently recommended to add `--exit-non-zero-on-fix` too per example in https://github.com/charliermarsh/ruff-pre-commit

* https://github.com/charliermarsh/ruff/pull/2668

Reply:
From the comments there:

> Pre-commit already knows if files changed

so it ought to be fine here...

Reply:
🤔 should be fine then